BookClub.Tests/Models/BookTest.cs

```diff
@@ -25,7 +26,86 @@ public void BookHasNecessaryProperties()
                     ISBN = "020161622X",
                     CoverUrl = "https://images-na.ssl-images-amazon.com/images/I/41BKx1AxQWL._SX258_BO1,204,203,200_.jpg"
                 };
+                var ValidationResult = new List<ValidationResult>();
+                var actual = Validator.TryValidateObject(book, new ValidationContext(book), ValidationResult, true);
+                Assert.True(actual);
             }
+
+            [Fact]
+            public void BookHasAnISBNWithMoreThanThirteenDigits()
+            {
+                var book = new DevDo.BookClub.Models.Book
+                {
+                    Title = "The Pragmatic Programmer",
+                    Author = "Andy Hunt",
+                    ISBN = "020161622X9409",
+                    CoverUrl = "https://images-na.ssl-images-amazon.com/images/I/41BKx1AxQWL._SX258_BO1,204,203,200_.jpg"
+                };
+                var ValidationResult = new List<ValidationResult>();
+                var actual = Validator.TryValidateObject(book, new ValidationContext(book), ValidationResult, true);
+                Assert.False(actual);
```

Comment:
Also, it would be nice to assert the validation message returned is the correct one